### PR TITLE
LPS-119818 Sort parameters

### PR DESCRIPTION
**Pull request by alinedoleron**
https://issues.liferay.com/browse/LPS-119818

### CODE REVIEW

**Comment by alinedoleron** [comment]
@natocesarrego, sorry for too many commits on this PR after the requested changes. I tried to merge for two times, but I was having too much problems so I sent new commits without merging.

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleBuilder/RulesSupport.es.js [hunk 1]
```diff
@@ -14,6 +14,9 @@
 
 import {PagesVisitor} from 'dynamic-data-mapping-form-renderer';
 
+import {DEFAULT_FIELD_NAME_REGEX, EMPTY_FIELD_REGEX} from '../../util/regex.es';
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleBuilder/RulesSupport.es.js:17]: @alinedoleron, why aren't we reusing it [here](https://github.com/liferay/liferay-portal/blob/7600a72838817db1adc56ba435ee7af1222873f7/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/LayoutProvider/LayoutProvider.es.js#L25) and [here](https://github.com/liferay/liferay-portal/blob/7600a72838817db1adc56ba435ee7af1222873f7/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/LayoutProvider/handlers/fieldEditedHandler.es.js#L22)?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/LayoutProvider/LayoutProvider.es.js [hunk 2]
```diff
@@ -322,10 +324,48 @@ class LayoutProvider extends Component {
...
+					)}
+				/>
 			</span>
 		);
 	}
 
+	_handleDeleteFieldModalButtonClicked(event) {
+		const {activePage, fieldName} = this.refs.existingRuleModal.data;
+
+		if (event.target.innerText == 'Delete') {
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/LayoutProvider/LayoutProvider.es.js:364]: @alinedoleron, please see https://github.com/natocesarrego/liferay-portal/pull/832#discussion_r491976005.

**Review by natocesarrego — COMMENTED** [review]
@alinedoleron, please check my comments below. 🙂 Almost there.

**Comment by natocesarrego** [comment]
@alinedoleron, if you have issues again during merge and commits organization, please reach out me to see if I can help you. 😉 

Thanks.

**Comment by alinedoleron** [comment]
new pr https://github.com/natocesarrego/liferay-portal/pull/839

**Pull request closed without merging**